### ISSUE TITLE
causal attention mask

### ISSUE THREAD

**Issue by itera-del** [hkproj/pytorch-llama#15]
Why is causal attention mask not used?
